src/pages/About/About.js

```diff
@@ -137,6 +136,12 @@ const About: ComponentType<Props> = ({ }: Props) => {
                 considered complete. Data for recent days are constantly being revised as more information becomes
                 available.
             </p>
+            <p className={'govuk-body govuk-body govuk-!-margin-bottom-8'}>
```

Comment:
Correction required:
-  Repetitive value of `govuk-body` for the `className` attribute.
- Only the last paragraph in a section - the one before the next heading - gets the classname `govuk-!-margin-bottom-8`.